src/CDEngineSDKTemplates/VsixBuilder/source.extension.vsixmanifest

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
     <Metadata>
-        <Identity Id="efb7398a-dc38-43ae-9963-e459cd84ae19" Version="5.106.0" Language="en-US" Publisher="C-Labs" />
-        <DisplayName>C-DEngine SDK Templates (5.106.0)</DisplayName>
+        <Identity Id="efb7398a-dc38-43ae-9963-e459cd84ae19" Version="5.107.0" Language="en-US" Publisher="C-Labs" />
+        <DisplayName>C-DEngine Visual Studio Templates (5.107.0)</DisplayName>
```

Comment:
should this not be 5.106.1 ? Its not July yet :)

Reply:
Depends when we release ;-) Ideally this should come from gitversion... Let me see if we can do this reasonably...

Reply:
Auto-versioned now.